packages/is-shallow-equal/index.js

```diff
@@ -0,0 +1,90 @@
+var isArray = Array.isArray,
+	keys = Object.keys;
```

Comment:
Are you using `var` for performance reasons? Thinking these may create ESlint errors if we add strict rules (const, let).

Reply:
>Are you using `var` for performance reasons? Thinking these may create ESlint errors if we add strict rules (const, let).

I'm using `var` because I'm effectively opting out of any transpilation. I think if we were to introduce such rules, we should apply them to each package's `src` folder, not necessarily a raw implementation as is done here.